org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/logger/AbstractLoggerSupport.java

```diff
@@ -40,6 +46,38 @@ public void setLogLevel(Logger logger, LogLevel level) {
     setLogLevel(assertNotNull(logger).getName(), level);
   }
 
+  protected void trackInitialState(String name) {
+    Map<String, Optional<LogLevel>> initialStateMap = m_initialStateMap;
+    if (initialStateMap != null) {
+      m_initialStateMap.putIfAbsent(name, Optional.ofNullable(getLogLevel(name)));
```

Comment:
m_initialStateMap could be null here if between line 51 and 52 another thread calls resetToInitialStates()
thy not use intialStateMap variable here?

Reply:
That's right - of course `initialStateMap` must be used here (otherwise this variable would not even be necessary).

Reply:
Fixed in 6e0a26a3aa4980807138223d90d4956d76d2a13a